qframe.go

```diff
@@ -1022,6 +1039,37 @@ func (qf QFrame) ToJSON(writer io.Writer) error {
 	return err
 }
 
+// ToSQL writes a QFrame into a SQL database.
+func (qf QFrame) ToSQL(tx *sql.Tx, table string) error {
+	if qf.Err != nil {
+		return errors.Propagate("ToSQL", qf.Err)
+	}
+	columns := qf.ColumnNames()
+	builders := make([]qfsqlio.ArgBuilder, len(columns))
+	for i, name := range columns {
+		builder, err := qfsqlio.NewArgBuilder(qf.columnsByName[name].Column)
+		if err != nil {
+			return err
+		}
+		builders[i] = builder
+	}
+	stmt, err := tx.Prepare(qfsqlio.Insert(table, columns))
+	if err != nil {
+		return err
+	}
+	for i, _ := range qf.index {
+		args := []interface{}{}
```

Comment:
A bit of a nitpick:
I've tried to be fairly consistent with initializing slices that have a known size upfront with the size (or capacity) using `make`. Sometimes it has a real performance impact (when working with slices holding column data for example). Sometimes it does not but then I'll still do it for consistency.
In this case I doubt it has any measurable performance impact but it would be nice to do it the same way across the code base.